README.md

```diff
@@ -33,17 +33,75 @@ style CDB fill:#ccf,stroke:#f66
 There are three modules for connection broker API. Theses module's states are enrolled, activated, deactivated, delisted.
```

Comment:
> There are three modules for connection broker API

I'd write down the module names here. Just knowing there are three of them isn't very useful on its own.
```
The connection broker API consists of three modules: Edge, Arbiter and Hub.
```

> Theses module's states are enrolled, activated, deactivated, delisted.

I think you're talking not about modules (the way the API is organized as you laid down in the previous sentence), but the broker instances themselves, which make more sense to have some state.

```
Each broker instance can find itself in one of the following states: enrolled, activated, deactivated, delisted.
```

(Perhaps elaborate what those states mean.)